docx/oxml/section.py

```diff
@@ -41,22 +45,24 @@ class CT_SectPr(BaseOxmlElement):
     """
     ``<w:sectPr>`` element, the container element for section properties.
     """
-    __child_sequence__ = (
-        'w:footnotePr', 'w:endnotePr', 'w:type', 'w:pgSz', 'w:pgMar',
-        'w:paperSrc', 'w:pgBorders', 'w:lnNumType', 'w:pgNumType', 'w:cols',
-        'w:formProt', 'w:vAlign', 'w:noEndnote', 'w:titlePg',
+    _tag_seq = (
+        'w:headerReference', 'w:footnotePr', 'w:endnotePr', 'w:type', 'w:pgSz',
+        'w:pgMar', 'w:paperSrc', 'w:pgBorders', 'w:lnNumType', 'w:pgNumType',
+        'w:cols', 'w:formProt', 'w:vAlign', 'w:noEndnote', 'w:titlePg',
         'w:textDirection', 'w:bidi', 'w:rtlGutter', 'w:docGrid',
         'w:printerSettings', 'w:sectPrChange',
     )
+    headerReference = ZeroOrMore('w:headerReference', successors=())
     type = ZeroOrOne('w:type', successors=(
-        __child_sequence__[__child_sequence__.index('w:type')+1:]
+        _tag_seq[_tag_seq.index('w:type')+1:]
     ))
     pgSz = ZeroOrOne('w:pgSz', successors=(
-        __child_sequence__[__child_sequence__.index('w:pgSz')+1:]
+        _tag_seq[_tag_seq.index('w:pgSz')+1:]
     ))
     pgMar = ZeroOrOne('w:pgMar', successors=(
-        __child_sequence__[__child_sequence__.index('w:pgMar')+1:]
+        _tag_seq[_tag_seq.index('w:pgMar')+1:]
     ))
+    del _tag_seq
```

Comment:
do you want me to list ALL the tag_seq attributes before this delete? Like on CT_Style

I think most of the attributes here in _tag_seq are OptionalAttributes. And so shouldn't be listed in this sequence thing? At least they aren't in CT_Style